### DISCUSSION TITLE
Does ImageMagick use WebSocket?

### DISCUSSION THREAD

**Discussion by pfischer66** [ImageMagick/ImageMagick#7769] [General] [answered]
Hello everybody,
I need help with a question from our IT-security: does ImageMagick use WebSocket?

Thanks for your support!

Peter Fischer

Phoenix Contact GmbH & Co. KG
Group Communications / System Operations
Flachsmarktstr. 8
32825 Blomberg / Germany

**Comment by dlemstra** [comment, marked as answer]
ImageMagick is a command line tool so the answer is no.
